R/convert_param.R

```diff
@@ -67,3 +67,47 @@ gamma_meansd2shapescale <- function(mean, sd) {
   scale <- sd^2 / mean
   list(shape = shape, scale = scale)
 }
+
+#' Converts the mean and standard deviation of the weibull distribution to the
+#' shape and scale parameterisation.
+#'
+#' @param mean The mean of the weibull distribution
+#' @param sd The standard deviation of the weibull distribution
+#'
+#' @return A named list with shape and scale parameters
+#' @export
+#'
+#' @examples
+#' weibull_meansd2shapescale(mean = 1.5, sd = 0.5)
+weibull_meansd2shapescale <- function(mean, sd) {
+
+  # give warning message about numerial inaccuracies
+  message("Numerical approximation used, results may be unreliable.")
```

Comment:
Would it be helpful to give users a sense of how precise / rough this approximation is?

Maybe something like this in the examples?

```r
wss <- weibull_meansd2shapescale(mean = 1.5, sd = 0.5)
weibull_shapescale2meansd(wss[["shape"]], wss[["scale"]])
```

Reply:
Resolved in 4e6213f8a4298d6cc8aeb74d206befe75e769d8c